cli/src/fmt/simple/payment_path.rs

```diff
@@ -0,0 +1,76 @@
+use super::Simple;
+use fmt::Render;
+use stellar_client::resources::PaymentPath;
+
+impl Render<PaymentPath> for Simple {
+    fn render(&self, payment_path: &PaymentPath) -> Option<String> {
+        let mut buf = String::new();
+        append!(
+            buf,
+            "Destination amount:               {}",
+            payment_path.destination_amount()
+        );
+        append!(
+            buf,
+            "Destination asset code:           {}",
+            payment_path.destination_asset().code()
```

Comment:
We have a `Render` implementation for `AssetIdentifier` now that gives the "standard" 1 liner: `XLM` or `<code>-<issuer>`. You could call `self.render(payment_path.destination_asset()).unwrap()` and not have to append each part individually.

Reply:
Nice catch! Updated